regexes/bots.yml

```diff
@@ -349,6 +362,14 @@
     name: 'Genieo'
     url: 'http://www.genieo.com'
 
+- regex: 'Gluten Free Crawler'
+  name: 'Gluten Free Crawler'
+  category: 'crawler'
+  url: 'http://glutenfreepleasure.com/'
+
+- regex: 'Guzzle'
+  url: 'https://github.com/guzzle/guzzle'
```

Comment:
Ok, did miss that one. Even it's/it'll be made up from the user agent.

What now? Should I add it into the file and make it another commit/request (maybe even with the tests) or will you add it on merge?